tests/integration/gemm/int4_dequantization_bias/main_dg2.cpp

```diff
@@ -500,22 +565,22 @@ void dequantize_gemm_run(int iter) {
             {bias_d, bias_add_shape}});
 
     typename gemm_op_t::template arguments_t<compute_policy::quant_type>
-            gemm_arg(matrix_m, matrix_k, matrix_n, A_d, matrix_k, B_d, matrix_n,
-                    C_d, matrix_n, scale_d, matrix_n, Acc_d, Cnt_d,
-                    epilogue_args);
+            gemm_arg(matrix_m, matrix_k, matrix_n, A_d, lda, B_d, ldb, C_d, ldc,
+                    scale_d, matrix_n, Acc_d, Cnt_d, epilogue_args);
 
     cl::sycl::nd_range<3> nd_range = gemm_op_t::get_nd_range(gemm_arg);
-    if (!gemm_op_t::can_implement(gemm_arg)) {
-        std::cout << "The arguments cannot be supported, aborting ... "
-                  << std::endl;
-        FAIL();
-    }
+    //     if (!gemm_op_t::can_implement(gemm_arg)) {
```

Comment:
emm, will we just delete this impl check in the future or fix the internal logic?

Reply:
In my experience, these limitations are by-design. Users should try scattered memory layout (or transposed layout) if `can_implement` returns false.